Source/WebCore/rendering/mathml/RenderMathMLBlock.cpp

```diff
@@ -275,6 +275,25 @@ void RenderMathMLBlock::styleDidChange(StyleDifference diff, const RenderStyle*
         setNeedsLayoutAndPrefWidthsRecalc();
 }
 
+void RenderMathMLBlock::handlePositionedAndFloatingChildren()
+{
+    for (auto* child = firstChildBox(); child; child = child->nextSiblingBox()) {
+        if (child->isOutOfFlowPositioned())
+            child->containingBlock()->insertPositionedObject(*child);
+
+        // According to the spec [1]:
+        // > The float property does not create floating of elements whose parent's computed display value is block math or inline math,
+        // > and does not take them out-of-flow.
+        // However, WebKit does not currently do this since `display: math` is unimplemented (see [2]).
+        // Since this leaves floats as neither positioned nor in-flow, this performs dummy layout to ensure that they are laid out at all.
+        // FIXME: Correctly handle floating children of computed display value `math` elements, and remove this.
+        // [1]: https://w3c.github.io/mathml-core/#css-styling
+        // [2]: https://bugs.webkit.org/show_bug.cgi?id=278533
```

Comment:
I don't know if adding [1] [2] references are quite common in WebKit code. In any case, maybe this could just be

```
// Perform dummy layout for floating children.
// FIXME: Per the spec, there should be no floating children inside MathML renderers. See webkit.org/b/278533
```

and all relevant information moved directly in the bug.

Reply:
Done